Pangram README claims input will only have ASCII

The README says:

    The alphabet used consists of ASCII letters `a` to `z`, inclusive, and is case
    insensitive. Input will not contain non-ASCII symbols.

There are German characters in one test which causes failures if not ignored or counted. But even then tests fail if code does not handle characters outside of ASCII. Specifically, Russian.

    --- FAIL: TestPangram (0.00s)
	pangram_test.go:37: Pangram test [Широкая электрификация южных губерний даст мощный толчок подъёму сельского хозяйства.], expected [false], actual [true]
	pangram_test.go:39: [Широкая электрификация южных губерний даст мощный толчок подъёму сельского хозяйства.] should not be a pangram because : Panagram in alphabet other than ASCII

I am OK with having to check if a rune is ASCII or not. But the README explicitly says not to worry about it. Or is 'symbols' here meant to mean things other than letters like braces, emoji, and the like? Either way the README is at best not clear and at worst wrong.

--- a/exercises/pangram/example.go
+++ b/exercises/pangram/example.go
@@ -4,7 +4,7 @@ import (
 	"strings"
 )
 
-const testVersion = 1
+const testVersion = 2
 
 func IsPangram(s string) bool {
 	lowerString := strings.ToLower(s)

--- a/exercises/pangram/pangram.go
+++ b/exercises/pangram/pangram.go
@@ -1,5 +1,5 @@
 package pangram
 
-const testVersion = 1
+const testVersion = 2
 
 func IsPangram(string) bool

--- a/exercises/pangram/pangram_test.go
+++ b/exercises/pangram/pangram_test.go
@@ -4,7 +4,7 @@ import (
 	"testing"
 )
 
-const targetTestVersion = 1
+const targetTestVersion = 2
 
 type testCase struct {
 	input         string
@@ -20,8 +20,6 @@ var testCases = []testCase{
 	{"the 1 quick brown fox jumps over the 2 lazy dogs", true, ""},
 	{"7h3 qu1ck brown fox jumps ov3r 7h3 lazy dog", false, "missing letters replaced by numbers"},
 	{"\"Five quacking Zephyrs jolt my wax bed.\"", true, ""},
-	{"Victor jagt zwölf Boxkämpfer quer über den großen Sylter Deich.", true, ""},
-	{"Широкая электрификация южных губерний даст мощный толчок подъёму сельского хозяйства.", false, "Panagram in alphabet other than ASCII"},
 }
 
 func TestTestVersion(t *testing.T) {

